app/code/Magento/InventoryLowStockNotification/view/adminhtml/layout/inventorylowstocknotification_lowstock_grid.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

Comment:
Maybe it is hard to implement
But my opinion we need to use UI component Grid instead of legacy grid based on blocks
We need to replace layout and declare new grid